### ISSUE TITLE
Provider is not importable without using replaces in go

### ISSUE THREAD

**Issue by 5-sigma** [ionos-cloud/terraform-provider-ionoscloud#168]
## Description
It is not possible to import the terraform provider into other go projects without using replace because the go.mod is missing a /vN (in accordance to it's respective major version)

## Expected behavior
https://github.com/ionos-cloud/terraform-provider-ionoscloud/blob/master/go.mod#L1
This line should end in /v6 for the latest release.

## How to Reproduce
Add provider to go.mod at v6.0.0 and do `go mod tidy`
Use this require:
```
require (
    github.com/ionos-cloud/terraform-provider-ionoscloud/v6 v6.0.0
)
```

## Error and Debug Output
```
go mod tidy
go: github.com/ionos-cloud/terraform-provider-ionoscloud/v6@v6.0.0: go.mod has non-.../v6 module path "github.com/ionos-cloud/terraform-provider-ionoscloud" (and .../v6/go.mod does not exist) at revision v6.0.0
```

**Comment by cristiGuranIonos** [comment]
We will add versioning, thanks!

**Comment by cristiGuranIonos** [comment]
Should be fixed in latest terraform release -  6.0.2
